README.md

```diff
@@ -20,7 +20,7 @@ vulnerability report.
 
 **IMPORTANT:**
 <br >Currently, Dependency Analytics only supports projects that use Maven (`mvn`), Node (`npm`), Golang (`go mod`) and
-Python (`pip`) ecosystems.
+Python (`pip`) ecosystems, and base images in `Dockerfile`.
```

Comment:
@ritz303 could you please review the doc changes for the dockerfile scanning feature?

Also, Itsik has some concerns on the docs which he found during testing. Could you please snyc with him?

Thanks.

Reply:
@xieshenzh : Sure, I'd be happy to take a look over your doc updates.  I did already submit [PR#154](https://github.com/redhat-developer/intellij-dependency-analytics/pull/154), adding the Docker scanning content to the IntelliJ docs.  This is the same content that I wrote for the VS Code docs regarding this feature.  How do you want to handle merging these doc updates?  I'm flexible, so just let me know whatever works easiest for you.

Regarding Itsik's concerns, I'll reach out to him.  Thanks for letting me know, much appreciated.

Reply:
I have cherry-picked your commits from #154 to this PR.
You can close #154. Thanks.

Reply:
Thank you! I've closed #154.